src/anomalib/models/efficient_ad/lightning_model.py

```diff
@@ -73,8 +73,8 @@ def __init__(
         lr: float = 0.0001,
         weight_decay: float = 0.00001,
         padding: bool = False,
-        pad_maps: bool = True,
         batch_size: int = 1,
+        pretraining_images_dir: str = "./datasets/imagenette",
```

Comment:
Would it make sense to make this parameter optional? That way, if it is `None` you can still do the automatic download of the Imagenet dataset.